liquibase-integration-tests/src/test/resources/liquibase/extension/testing/command/calculateChecksum.test.groovy

```diff
@@ -47,7 +47,7 @@ Optional Args:
         ]
 
         expectedResults = [
-                checksumResult   : "9:10de8cd690aed1d88d837cbe555d1684"
+                checksumResult   : "9:99091c432814de759be3e5f65eececf0"
```

Comment:
I think these test data updates are not correct because this generation of a new Checksum value is because of the new property included in `createTable` change, which at the moment is not an expected property there. In my opinion, we should revert this. 

Thanks,
Daniel.

Reply:
OK, I reverted the changes that brought `preserverNullValues` to the surface.
Now that property is never serialised.  It does appear in the meta-data, like `serializedFields`.